docs/guides/api-gateway/kubernetes.mdx

```diff
@@ -1,430 +1,558 @@
+    domain](/docs/guides/other-guides/how-to-set-up-a-custom-domain), like
+    `https://api.example.com`.
+  - We'll refer to this domain as `<YOUR_NGROK_DOMAIN>`.
+- An [**Auth0
+  account**](https://auth0.com/signup?place=header&type=button&text=sign%20up)
+  (free is fine) for creating an API and generating JWTs.
+
+### Deploy demo API services
+
+If you want to quickly wire up a POC using ngrok, we recommend using our [demo
+API service](https://github.com/ngrok-samples/api-demo), which responds with details
+about your requests.
+
+You can also bring your own services to this guide—in that case, make sure to
+change service names, target ports, and namespaces accordingly in the resources
+you'll define in the next few steps.
+
+The two manifests below provision pods and resources for `abc` on
+and `xyz`, respectively, and configure the container's environment variables to
+respond uniquely to "prove" you're routing to multiple services.
+
+<Tabs>
+  <TabItem value="Deployment+service: abc">
+
+```yaml title="abc.yaml" mode=file
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: abc
+spec:
+  selector:
+    matchLabels:
+      app: abc
+  template:
+    metadata:
+      labels:
+        app: abc
+    spec:
+      containers:
+        - name: abc
+          image: joelatngrok/api-demo:latest
+          ports:
+            - containerPort: 4000
+          env:
+            - name: MESSAGE
+              value: "Hello from service abc!"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: abc-service
+spec:
+  selector:
+    app: abc
+  ports:
+    - port: 4000
+      targetPort: 4000
 ```
 
-## **Step 1**. Apply the ngrok Kubernetes Operator {#apply-kubernetes-operator}
+  </TabItem>
+  <TabItem value="Deployment+service: xyz">
 
-Add the [ngrok Kubernetes Operator](https://github.com/ngrok/ngrok-operator)
-repo to Helm.
-
-```bash
-helm repo add ngrok https://charts.ngrok.com
-helm repo update
+```yaml title="xyz-.yaml" mode=file
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: xyz
+spec:
+  selector:
+    matchLabels:
+      app: xyz
+  template:
+    metadata:
+      labels:
+        app: xyz
+    spec:
+      containers:
+        - name: xyz
+          image: joelatngrok/api-demo:latest
+          ports:
+            - containerPort: 5000
+          env:
+            - name: MESSAGE
+              value: "Hello from service xyz!"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: xyz-service
+spec:
+  selector:
+    app: xyz
+  ports:
+    - port: 5000
+      targetPort: 5000
 ```
 
-Apply the latest Gateway API CRDs to your cluster.
+  </TabItem>
+</Tabs>
+
+Apply these with `kubectl apply -f ...` into your cluster.
+
+## 1. Deploy the ngrok Kubernetes Operator
 
-```bash
-kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
+Add the ngrok Kubernetes Operator repo to Helm.
```

Comment:
suggestion: maybe add a note directing users to the more detailed installation guide for the operator since we have a page for that.